modules/users/client/views/profile/profile-view-basics.client.view.html

```diff
@@ -7,3 +7,8 @@
   ng-if="::app.user._id !== profileCtrl.profile._id"
   username="::profileCtrl.profile.username"
 ></report-member-link>
+<br>
```

Comment:
These links felt a bit close and hard to tap with big fingers on mobile. Let's try a bigger space with `<br/><br/>`?